pom.xml

```diff
@@ -136,6 +136,9 @@
         </profile>
         <profile>
             <id>spark310tests</id>
+            <modules>
+                <module>tests-next-spark</module>
```

Comment:
I named it so its not tied to 3.1.0 only. If you can come up with a better name please let me know

Reply:
I personally would prefer to have the name of the version in it, something like `tests-spark310-plus` to indicate that this is for spark 3.1.0 and above.  The only time we could get away with a `tests-next` type package name would be if we have one and only one set of tests that are specific to a version.  And even then I don't see much advantage to not being specific in the name.

Reply:
Any objections to `tests-spark310+`